EventFilter/EcalRawToDigi/BuildFile.xml

```diff
@@ -16,11 +14,15 @@
 <use name="FWCore/PluginManager"/>
 <use name="Geometry/EcalMapping"/>
 <use name="Geometry/Records"/>
-<use name="HeterogeneousCore/CUDACore"/>
-<use name="HeterogeneousCore/CUDAUtilities"/>
 <use name="RecoLocalCalo/EcalRecAlgos"/>
 <use name="RecoLocalCalo/EcalRecProducers"/>
 <use name="Utilities/StorageFactory"/>
+
+<use   name="cuda"/>
+<use   name="HeterogeneousCore/CUDAUtilities"/>
+<use   name="HeterogeneousCore/CUDACore"/>
+<use   name="CUDADataFormats/EcalDigi" />
+
```

Comment:
can you revert the changes to this file ?
all `<use .../>` were already there.

Reply:
Done, and it will come in the next commit.